js_tests/gis/mapwidget.test.js

```diff
@@ -0,0 +1,46 @@
+/* global module, test */
+/* eslint global-strict: 0, strict: 0 */
+'use strict';
+
+module('gis.OLMapWidget');
+
+test('MapWidget.featureAdded', function(assert) {
+    var options = {id: 'id_point', map_id: 'id_point_map', geom_name: 'Point'};
+    var widget = new MapWidget(options);
+    assert.equal(widget.layers.vector.features.length, 1);
+    assert.equal(
+        widget.layers.vector.features[0].geometry.toString(),
+        'POINT(7.8177 47.397)',
+        'Point addded to vector layer'
+    );
+});
+
+test('MapWidget.map_srid', function(assert) {
+    var options = {id: 'id_point', map_id: 'id_point_map', geom_name: 'Point'};
+    var widget = new MapWidget(options);
+    assert.equal(widget.options['map_srid'], 4326, 'SRID 4326');
```

Comment:
We prefer writing properties using dot notation when it's valid to do so (so `widget.options.map_srid` instead of `widget.options['map_srid']`).  This is specified in the ESLint configuration, but it looks this directory isn't currently linted by the pretest command in `package.json`.